gist/client.py

```diff
@@ -227,6 +230,30 @@ def elide(txt, width=terminal_width()):
     return txt
 
 
+def get_value_from_command(value):
+    """Return the value of a config option, potentially by running a command
+
+    When a config option begins with a ``!`` interpret the remaining text as a
+    shell command which when run prints the config option value to stdout.
+    Otherwise return the original string.
+
+    Argument:
+        string: value of an option returned from the config file.
```

Comment:
Sorry, this needs to be updated to match the argument.

Reply:
Ah yes, should be fixed.